### PR TITLE
404Page 

### PR DESCRIPTION
**Pull request by anoopgcmz**

### CODE REVIEW

File: app/views/404.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  </div>
+
+  <div class="login-form padded">
+    <img src="images/adminium_2.png" class="img-responsive login-title" width="347">
+    <div class="text-center">
+      <span class="pnfmsg">Page Not Found</span>
+      <!-- <br/>
+      <h4>Page may be removed or the link dosen't exist</h4> -->
+      <h4>
+        <a href="javascript:void(0);" ng-click="goBack()" class="pnfcolor"> Click Here To Go Back </a>
```
Comment (gchablowski) [inline app/views/404.html]: the javascript:void(0); is not necessary

File: test/e2e/done/404-view.spec.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+describe('rates page', function () {
+
+    var module = require('./module/module.js');
+
+    beforeAll(module.login);
+    // afterAll(module.logout);
```
Comment (gchablowski) [inline test/e2e/done/404-view.spec.js]: it will never logout

File: app/views/404.html (hunk 1 above)
Comment (anoopgcmz) [reply app/views/404.html]: Updated with a button

File: test/e2e/done/404-view.spec.js (hunk 2 above)
Comment (anoopgcmz) [reply test/e2e/done/404-view.spec.js]: Uncommented afterAll

**Pull request merged**